include/rwsdk/rwplcore.h

```diff
@@ -673,6 +673,29 @@ struct RwMatrixTag
     // Offset: 0x30
     RwV3d pos;
     RwUInt32 pad3;
+
+	RwMatrixTag& operator=(const RwMatrixTag& rhs)
```

Comment:
I'm not so sure about this. The S32 casts look like they're necessary because the asm is likely a compiler generated memcpy. This happens when the compiler generates the assignment function implicitly, meaning it is not very declared or defined.

Additionally render ware is a C library AFAIK so it wouldn't have supported this feature.

What happens when you remove this entirely?